src/dom/html/WindowRe.re

```diff
@@ -95,6 +95,28 @@ module Impl (T: { type t; }) => {
 
 
   external setOnLoad : t_window => (unit => unit) => unit = "onload" [@@bs.set]; /* use addEventListener instead? */
+
+  module Image = {
+    type t;
+    external make : unit => t = "Image" [@@bs.new];
+    external src : t => string => unit = "" [@@bs.set];
+    external getSrc : t => string = "src" [@@bs.get];
+    external onload : t => (unit => unit) => unit = "" [@@bs.set];
+    external naturalHeight : t => string = "" [@@bs.get];
+    external naturalWidth : t => string = "" [@@bs.get];
+  };
+  
+  module File = {
+    type t;
+    external _type : t => string = "type" [@@bs.get];
+    external preview : t => string = "" [@@bs.get];
+  };
+  
+  module URL = {
```

Comment:
Similarly, this API is not part of the DOM either. As joy would have it, it's part of both the `FIle API` and `URL` specs, and these function specifically are introduced in the File API, so perhaps put these in `src/file/` too.